__test__/api/comments/createSingleComment.test.js

```diff
@@ -40,8 +40,6 @@ describe("POST /comments", () => {
     };
     expect(res.status).toBe(201);
     expect(res.body.status).toEqual("success");
-    expect(expected.applicationId).toEqual(global.application._id);
```

Comment:
was this an error, deleting from another file..?
this change is not required

Reply:
Im on this now